src/java/picard/vcf/LiftoverVcf.java

```diff
@@ -281,6 +311,7 @@ public static void main(final String[] args) {
                 else {
                     sorter.add(builder.make());
                 }
+                reverseComplementAlleleMap.clear();
```

Comment:
Nope, just tested it, memory goes up to 8G and it looks like GC starts to take over.